compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -445,6 +444,7 @@ class ValuePropagation : public TR::Optimization
    void removeRestOfBlock();
    void mustTakeException();
    void processTrees(TR::TreeTop *startTree, TR::TreeTop *endTree);
+   virtual void transformRecognizedMethod(TR::Node *node) {}
```

Comment:
This will be called near the beginning of constrainCall in VPHandlers.cpp, together with a few other lines of change, as part of my vp opt. I guess I should leave them to a different PR.